frontend/src/components/StateTooltip.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import IconButton from '@material-ui/core/IconButton';
+import Tooltip from '@material-ui/core/Tooltip';
+import InfoIcon from '@material-ui/icons/Info';
+import { DOCS } from '../constants';
+
+class StateTooltip extends React.Component {
+    constructor(props) {
+        super(props);
+    }
+
+    render() {
+        const title = (
+            <div>
+                <div>State 1: uploading</div>
+                <div>State 2: created</div>
+                <div>State 3: staged</div>
+                <div>State 4: making</div>
```

Comment:
'making' only applies to make bundles, so we should have switch logic that determines the set of states based on the bundle type.